patches/minecraft/net/minecraft/server/dedicated/DedicatedServer.java.patch

```diff
@@ -1,46 +1,56 @@
 --- a/net/minecraft/server/dedicated/DedicatedServer.java
 +++ b/net/minecraft/server/dedicated/DedicatedServer.java
-@@ -97,6 +97,7 @@
+@@ -15,6 +15,7 @@
+ import java.net.InetAddress;
+ import java.net.Proxy;
+ import java.nio.charset.StandardCharsets;
++import java.nio.file.Paths;
```

Comment:
Import in patch file

Reply:
It's used on line [25](https://github.com/MinecraftForge/MinecraftForge/blob/77a55ae3180a22ee698d7395ec800a9fd2a4d992/patches/minecraft/net/minecraft/server/dedicated/DedicatedServer.java.patch#L25)

Reply:
Sorry, I missed that. Either way, patches shouldn't contain imports. You should always directly reference a class in patches, if it's not already imported, to minimize patch size. So line 25 needs to be `this.field_71340_o.func_219033_a(properties -> ServerProperties.func_218985_a(java.nio.file.Paths.get("server.properties")));`

Reply:
Regardless of if it’s used or not, patch files should NEVER have imports, always use Fully Qualified Names instead.

Reply:
Okedoke, thanks for clarifying